model/firewall.rb

```diff
@@ -17,7 +17,7 @@ def insert_firewall_rule(cidr, port_range)
       port_range: port_range
     )
 
-    vm&.incr_update_firewall_rules
+    private_subnet&.incr_update_firewall_rules
```

Comment:
(Note, no need to fix with current PR) It might cause connection problems for the new VMs to be created in an existing subnet. We already discussed it offline and it will be fixed with https://github.com/ubicloud/ubicloud/pull/1463